site/vite.config.mts

```diff
@@ -28,6 +29,19 @@ export default defineConfig({
 		emptyOutDir: false,
 		// 'hidden' works like true except that the corresponding sourcemap comments in the bundled files are suppressed
 		sourcemap: "hidden",
+		rollupOptions: {
```

Comment:
Can we push site changes in a separate PR? perhaps somebody with a frontend skillset can review them once there is a feature code ready.

Reply:
https://github.com/coder/coder/pull/17123